src/Sulu/Bundle/LocationBundle/Resources/views/Template/content-types/location.html.twig

```diff
@@ -7,8 +7,8 @@
      data-type-instance-name="{{ id|raw }}"
      data-instance-name="{{ id|raw }}"
      data-form="true"
-     data-aura-countries="{{ params.countries|json_encode }}"
-     data-aura-map-providers="{{ params.mapProviders|json_encode }}"
+     data-aura-countries="{{ parameterToKeyValue(params.countries.value)|json_encode }}"
```

Comment:
That was a more general question, shouldn't the parameterToKeyValue take a localization as well?